storage/container_storage_interface/persistent-storage-csi-cinder.adoc

```diff
@@ -18,6 +18,16 @@ To create CSI-provisioned PVs that mount to OpenStack Cinder storage assets, {pr
 * The _OpenStack Cinder CSI driver_ enables you to create and mount OpenStack Cinder PVs.
 
 include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+{product-title} defaults to using an in-tree (non-CSI) plug-in to provision Cinder storage.
+
+Over subsequent {product-title} versions, volumes provisioned using existing in-tree plug-ins are planned for migration to their equivalent CSI driver. CSI automatic migration should be seamless. Migration does not change how you use all existing API objects (for example, PersistentVolumes, PersistentVolumeClaims, and StorageClasses). For more information about migration, see xref:../../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration].
```

Comment:
(Applies to each instance of this admonition)

Per IBM SG's "Inline examples" section, consider something like:

> Migration does not change how you use all existing API objects, such as `PersistentVolume`, `PersistentVolumeClaim`, and `StorageClass` objects.

Note also in the above suggestion the API object formatting per [repo docs guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting).